src/frontend/src/lib/i18n/en.json

```diff
@@ -264,7 +264,26 @@
 		"data": "Data",
 		"no_match": "No document matches your filters.",
 		"delete": "Delete document?",
-		"description": "Description"
+		"description": "Description",
+		"document_submission_success": "Document has been successfully submitted!",
+		"document_submission_failed": "Document submission failed!",
+		"field_doc_id_label": "Doc ID",
```

Comment:
maybe we can create a new group `document_field` at the same level of `document`?

```
"document: {
   ....
},
"document_field": {
   ...
```

That way the new keys are a bit more scoped